tests/unit/amplitude/test_yaml_helicity.py

```diff
@@ -91,32 +91,27 @@ def test_kinematics_section(imported_dict):
 
 def test_parameter_section(imported_dict):
     parameter_list = imported_dict["Parameters"]
-    assert len(parameter_list) == 6
+    assert len(parameter_list) == 12
     for parameter in parameter_list:
         assert "Name" in parameter
         assert "Value" in parameter
-        assert "Type" in parameter
         assert parameter.get("Fix", True)
 
 
 def test_dynamics_section(imported_dict):
     dynamics = imported_dict["Dynamics"]
-    assert len(dynamics) == 1
+    assert len(dynamics) == 3
 
     j_psi = dynamics["J/psi(1S)"]
     assert j_psi["Type"] == "NonDynamic"
     assert j_psi["FormFactor"]["Type"] == "BlattWeisskopf"
-    assert j_psi["FormFactor"]["MesonRadius"] == 1.0
+    assert j_psi["FormFactor"]["MesonRadius"] == "MesonRadius_J/psi(1S)"
```

Comment:
Test the value as well?

Reply:
Afaiu that should be available through the fit parameters section now, using that name right?

Reply:
Sorry, overlooked this comment. Yes actually it was a bug: the meson radius from the fit parameters section should have been referenced, not a hard-coded value.

Reply:
so the additional test im proposing would make sure that the fit parameter is also registered correctly

Reply:
Well the tests need a lot more improvements anyway. So I leave it up to you if you want to add anything to this.

Reply:
ad7e2d7f
But I indeed doubt this test adds much